studio-docs/source/metrics/field-usage.mdx

```diff
@@ -94,3 +96,59 @@ A referenced field might also not be resolved for any of these reasons:
 * The resolver is skipped due to a `@skip` or `@include` directive.
 * The field is part of a fragment that doesn't apply to a particular object.
 * The field's resolver is called for each item in a list, but the list is empty.
+
+### `@key` and `@requires` fields in a federated graph
+
+> This case applies only to graphs that use [Apollo Federation](https://www.apollographql.com/docs/federation).
+
+Let's say our federated graph includes these two subgraph schemas:
+
+<CodeColumns>
+
+```graphql
+# Products subgraph
+
+type Product @key(fields: "id") {
+  id: ID!
+  name: String!
+}
+```
+
+```graphql
+# Reviews subgraph
+
+extend type Product @key(fields: "id") {
+  id: ID! @external
+}
+
+type Review {
+  id: ID!
+  score: Int!
+  product: Product!
+}
+
+type Query {
+  reviews: [Review!]!
+}
+```
+
+</CodeColumns>
+
+Now, let's say a client executes the following query against the gateway:
+
+```graphql
+query GetAllReviews {
```

Comment:
I'm assuming you actually tested out these subgraphs and operation (eg in workbench)?